### DISCUSSION TITLE
Welcome to lv2tutorial Discussions!

### DISCUSSION THREAD

**Discussion by sjaehn** [sjaehn/lv2tutorial#17] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by eventual-recluse** [comment]
Greetings sjaehn & everyone. Thanks for these tutorials - they are excellent and have been extremely helpful for getting into lv2 programming. I enjoy making music and try to use FOSS as much as possible, so would be very happy to see more lv2 plugins. LV2 also seems to be a very reliable format as far as I can tell from the perspective of a user.

These tutorials really fill a gap when it comes to information on the web about programming lv2. The example plugins at https://lv2plug.in/book/ are useful, but it helps enormously to get more of an explanation of what's going on. And I really appreciate the crash course in C and C++. I must admit that I learned a few things there!

One thing that I think would be really useful:  an example plugin which implements a buffer of some kind, like a simple delay. This seems to be one thing missing in the samples out there.

Cheers

**Reply by sjaehn** [reply]
Thanks. And good idea. Delays with (ring) buffers are easy to make. This could be something for a third tutorial series. About some important DSP algorithms.

**Comment by eventual-recluse** [comment]
Sounds good. Keep up the good work!
